modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -0,0 +1,320 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sts-creating-cluster-customizations-ocm_{context}"]
+= Creating a cluster through {cluster-manager} using customizations
+
+When you create a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can customize your installation interactively through {cluster-manager-first}.
+
+You can also choose between the `auto` and `manual` modes when creating the required STS resources.
+
+[IMPORTANT]
+====
+Only public and AWS PrivateLink clusters are supported with STS. Regular private clusters (non-PrivateLink) are not available for use with STS.
+====
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installations.
+====
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You verified that the AWS Elastic Load Balancing (ELB) service role exists in your AWS account.
++
+[NOTE]
```

Comment:
Can you please also implement any feedback that applies from the other large procedure to this one?

Reply:
Resolved.